programs/github

```diff
@@ -59,4 +126,26 @@ if action == 'clone' then
 	term.scroll(1)
 	return
 end
-print(usage)
+
+if action == 'auth' then
+	local user, token = args[1], args[2]
+	if not user or not token then return printError("No user or token specified.") end
```

Comment:
This should also verify the delete argument wasn't specified, so a user can simply do `auth -D aaronmallen` and the authentication data will be deleted without having to specify `user` and `token`.